mailjet_simple.module

```diff
@@ -267,5 +267,56 @@ function mailjet_simple_settings_defaults() {
     'public_key' => '',
     'private_key' => '',
     'lists' => [],
+    'additional_settings' => array(
+      'cancel_delete' => TRUE,
+    ),
   ];
 }
+
+/**
+ * Implements hook_user_delete().
+ */
+function mailjet_simple_user_delete($account) {
+  mailjet_simple_unsubscribe_all_lists($account);
```

Comment:
The settings check needs to happen in each of these hooks and not in the mailjet_simple_unsubscribe_all_lists() method. This will allow that method to be called w/o the setting in other use cases. 

Otherwise, looks good!